prow/config/branch_protection.go

```diff
@@ -53,6 +53,9 @@ type Policy struct {
 	// Exclude specifies a set of regular expressions which identify branches
 	// that should be excluded from the protection policy
 	Exclude []string `json:"exclude,omitempty"`
+	// Include specifies a set of regular expressions which identify branches
+	// that should be included from the protection policy
+	Include []string `json:"include,omitempty"`
```

Comment:
Please document that this is mutually exclusive with `Exclude` and add a check into the config load that errors if both are set